### PR TITLE
419-test-outliering-with-anon-data

### PR DESCRIPTION
**Pull request by giuliag92**
# Summary

Add your summary here - keep it brief, to the point, and in plain English.

# Checklists

This pull request meets the following requirements:

- [x] installable with all dependencies recorded
- [x] runs without error
- [x] follows PEP8 and project specific conventions
- [x] appropriate use of comments, for example no descriptive comments
- [x] functions documented using Numpy style docstings
- [x] assumptions and decisions log considered and updated if appropriate
- [x] unit tests have been updated to cover essential functionality for a reasonable range of inputs and conditions
- [x] other forms of testing such as end-to-end and user-interface testing have been considered and updated as required
- [x] tests suite passes (locally as a minimum)
- [x] peer reviewed with review recorded

If you feel some of these conditions do not apply for this pull request, please
add a comment to explain why.

### CODE REVIEW

File: mbs_results/calculate_winsorised_weight.py [hunk 1]
```diff
@@ -1,21 +1,20 @@
 import numpy as np
-import pandas as pd
 
 
 def calculate_winsorised_weight(
```
Comment (AntonZogk) [inline mbs_results/calculate_winsorised_weight.py:4]: Can we please update the docstrings to match the arguments order ?

File: mbs_results/calculate_ratio_estimation.py [hunk 2]
```diff
@@ -1,24 +1,20 @@
-import pandas as pd
+import numpy as np
 
 
 def calculate_ratio_estimation(
```
Comment (AntonZogk) [inline mbs_results/calculate_ratio_estimation.py:4]: Can we please update docstrings ? 11 args in docstrings but 9 actual args, ( i think the order is fine)

**Review by AntonZogk — APPROVED** [review]
Amazing work !!! It ran smoothly :) !!! Just 2 minor comments for the docstrings !!!

**Comment by giuliag92** [comment]
Docstrings fixed, thank you Anton!!! :)

**Pull request merged**